app/src/components/main/hexColorPicker/hexColorPicker.jsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { HexColorPicker, HexColorInput } from 'react-colorful';
+import classNames from 'classnames/bind';
+import styles from './hexColorPicker.scss';
+
+const cx = classNames.bind(styles);
+
+export const HexColorPickerComponent = ({ label, color, presets, onChange }) => {
+  const [currentColor, setCurrentColor] = useState(color);
+
+  useEffect(() => onChange(currentColor), [currentColor]);
+
+  useEffect(() => setCurrentColor(color), [color]);
```

Comment:
Why are you duplicate prop `color` in component's state?
Why not to use the props `color` and `onChage` directly where it is needed?

Reply:
This is how it was done firstly, but it didn't work